### PR TITLE
Project html files

### PR DESCRIPTION
**Pull request by RohitAmdahl**
i have installed Bootstrap , SASS and Liver server.
 I have used to create social media website project CSS frameworks bootstrap and sass and i have customized according to my design.
 I have done my best to understand the bootstrap and i have used bootstrap class but also i have made my own classes, i have used components from bootstrap and changed UI  according to my project.  i have made all file separate all Sass files is in folder called partials.

### CODE REVIEW

**Comment by Fermain** [comment]
Please check the issues tab and resolve these before the review can start.

File: login.html [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+                />
+              </div>
+              <div class="password">
+                <label for="floatingPassword">Password</label>
+                <input
+                  type="password"
+                  id="Passwordinput"
+                  class="form-control"
+                  placeholder="write your password"
+                  maxlength="10"
```
Comment (Fermain) [inline login.html]:
```suggestion
                  minlength="8"
```

It's considered bad form to set a max length for passwords, instead there should be a min length. Check out this article about the subject: https://specopssoft.com/blog/password-length-best-practices

Comment (RohitAmdahl) [reply login.html]: i was thinking it about it but i got confuse by that why min length and why max length so i chose choose max length, thank i will go to that steps

File: index.html [hunk 2]
```diff
@@ -4,11 +4,200 @@
...
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </main>
+    <!-- main finished here footer section start from her  -->
+    <footer>
+      <div class="container">
+        <div class="row logo">
```
Comment (Fermain) [inline index.html:152]: The child of a row should always be a column class

File: profile.html [hunk 3]
```diff
@@ -0,0 +1,383 @@
...
+              </div>
+            </div>
+          </div>
+        </div>
+        <!-- next section is  post feature -->
+        <div class="container">
+          <div class="row">
+            <div class="center">
+              <div class="col-12 col-sm-12 col-md-8">
+                <form class="post">
```
Comment (Fermain) [inline profile.html]: Form validation + form action would be a good improvement here

File: src/scss/partials/_body.scss [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+
+body {
+  font-family: $font-family;
+  background: $primary-color;
+}
+
+footer {
+  margin: 0 auto;
+  width: 100%;
+  max-width: 250px;
```
Comment (Fermain) [inline src/scss/partials/_body.scss]: If you use the row/col system correctly here you won't need max-width

File: src/scss/partials/_body.scss [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+  padding-top: 35px;
```
Comment (Fermain) [inline src/scss/partials/_body.scss]: `p-` classes for padding. Don't hard code padding values when a utility class can do it already.

File: src/scss/partials/_form.scss [hunk 6]
```diff
@@ -0,0 +1,33 @@
+.login_pic {
+  width: 100%;
+  max-width: 500px;
+  padding: 25px;
```
Comment (Fermain) [inline src/scss/partials/_form.scss]: Same here, use a padding class.

File: src/scss/partials/_form.scss [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+}
+
+p {
+  width: 100%;
+  max-width: 750px;
+  margin: 0 auto;
```
Comment (Fermain) [inline src/scss/partials/_form.scss]: `.mx-auto`

File: src/scss/partials/_form.scss [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+}
+
+.mb-4 {
+  padding: 10px;
```
Comment (Fermain) [inline src/scss/partials/_form.scss]: This is a big problem. Margin classes are for margins only. Not padding, that's what padding classes are for.

File: src/scss/partials/_form.scss [hunk 9]
```diff
@@ -0,0 +1,33 @@
...
+}
+
+#Emailinput {
+  width: 100%;
```
Comment (Fermain) [inline src/scss/partials/_form.scss]: `.w-100`

File: src/scss/partials/_nav.scss [hunk 10]
```diff
@@ -0,0 +1,104 @@
+$green: #9cdad7;
```
Comment (Fermain) [inline src/scss/partials/_nav.scss]: All variables belong in the `_variables.scss` file.

File: src/scss/partials/_nav.scss [hunk 11]
```diff
@@ -0,0 +1,104 @@
...
+$background_btn: #9cdad7;
+$text-color: #373737;
+$blue: #ace7ef;
+$background_btn_hover: linear-gradient(120deg, #e0c3fc 0%, #8ec5fc 100%);
+
+.wrapper {
+  display: flex;
```
Comment (Fermain) [inline src/scss/partials/_nav.scss]: Use `.d-flex`

File: src/scss/partials/_nav.scss [hunk 12]
```diff
@@ -0,0 +1,104 @@
...
+  list-style: none;
```
Comment (Fermain) [inline src/scss/partials/_nav.scss]: There's also a bootstrap class for this

**Review by Fermain — CHANGES_REQUESTED** [review]
This is mostly there, but you can reduce the amount of custom SCSS you have written by a considerable amount by using utility classes more. My comments are not exhaustive, but as a rule, you should not have padding, margin or display, text-transform, text-decoration styles in your code as this is all handled well by bootstrap already.

File: src/scss/partials/_form.scss (hunk 6 above)
Comment (RohitAmdahl) [reply src/scss/partials/_form.scss]: thanks i will do it

File: src/scss/partials/_body.scss (hunk 5 above)
Comment (RohitAmdahl) [reply src/scss/partials/_body.scss]: yes i will do the changes

File: src/scss/partials/_form.scss (hunk 8 above)
Comment (RohitAmdahl) [reply src/scss/partials/_form.scss]: i have changed

File: index.html (hunk 2 above)
Comment (RohitAmdahl) [reply index.html:152]: yes i changed it and added div and class col

File: profile.html (hunk 3 above)
Comment (RohitAmdahl) [reply profile.html]: i will  change now

File: src/scss/partials/_body.scss (hunk 4 above)
Comment (RohitAmdahl) [reply src/scss/partials/_body.scss]: yes i added div and class col in my all 3 pages with width class and padding and margin class

File: src/scss/partials/_nav.scss (hunk 10 above)
Comment (RohitAmdahl) [reply src/scss/partials/_nav.scss]: if I move this colour I am getting an error that is not defined.

File: src/scss/partials/_form.scss (hunk 9 above)
Comment (RohitAmdahl) [reply src/scss/partials/_form.scss]: changed

File: src/scss/partials/_form.scss (hunk 7 above)
Comment (RohitAmdahl) [reply src/scss/partials/_form.scss]: changed

File: src/scss/partials/_form.scss (hunk 8 above)
Comment (RohitAmdahl) [reply src/scss/partials/_form.scss]: i have done same changes in log in page

**Comment by RohitAmdahl** [comment]
liver server setting .json {
  "liveServer.settings.port": 5501
} 
do i need to change this? I am not getting it

File: src/scss/partials/_nav.scss (hunk 10 above)
Comment (Fermain) [reply src/scss/partials/_nav.scss]: You need to `@import "variables";`

**Comment by Fermain** [comment]
> liver server setting .json { "liveServer.settings.port": 5501 } do i need to change this? I am not getting it

It's not important

File: src/scss/partials/_nav.scss (hunk 10 above)
Comment (RohitAmdahl) [reply src/scss/partials/_nav.scss]: done

**Review by Fermain — APPROVED** [review]
This is a good submission with a really creative theme. The requirements have been satisfied 😄

**Comment by RohitAmdahl** [comment]
I have merged  branches

**Pull request merged**